### ISSUE TITLE
ESET Detection

### ISSUE THREAD

**Issue by C-Noob404** [helldivers2fixes/HellbombScript#25]
Hi all,

I love your script, and I think it saved me before. But since I have daily crashes again since the latest patch, I've discovered a problem.
The script detects ESET on my PC, but I never had ESET installed.
Can you tell me what this could be about?

Thank you very much an have a nice Weekend.

**Comment by helldivers2fixes** [comment]
TL;DR, you're probably fine. If you're crashing alot now, and you're using the SPEAR, that may be the explantion.

**Why You're Fine:**
Do you have a program that has the letters "eset" in the name? For example: a program with the name **Ready, Reset!** would get detected as ESET.
This is because the programmer is an idiot and was trying to cast a wide net to make sure to detect various versions of programs. The risk of course is in what is happening to you (false positives).

There's the [relevant line of code](https://github.com/helldivers2fixes/HellbombScript/blob/87c6868a56a01d32b6e19c9bcb84ad4e3610d2d4/Hellbomb%20Script.ps1#L204). You can see I do a wild card match so it's looking for anything that's *ESET* which means that it could detect ANYTHING with eset in it.

**Comment by helldivers2fixes** [comment]
Addressed with a information message in Commit: 207899eaa3029c4f64073a8d093f70b60ed42453
